### PR TITLE
[WIP] Documentation review and update 

### PR DESCRIPTION
**Pull request by line47**
**WIP - DO NOT MERGE**

This PR is aimed at re-evaluating the documentation here in the repo. What should stay, what needs to be updated and or changed. 

### Added
- Criteria page which explains how to propose new components as well as how the process of getting them published works.
- Criteria page to give insight into what criteria should be met for introducing a new component, pattern or style.

### Changed
- Moved wiki pages into a guides folder for easier review, updating and editing. 
- Moved guidance pages from the root to the guides folder.
- Moved the running local information to the README as this is a typical convention.
- Updated the contributing file to be more clear and direct about the process folks should follow to contribute to the Design System.
- Updated the governance document to make a bit more generic

#### To do 
- [ ] After this PR is merged Remove wiki pages.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,8 +29,63 @@ Examples of the design system in use can be found in the [`examples` directory](
...
+If you have multiple directories inside of `packages/themes`, you can specify which theme to use by passing the scripts the name of the folder. For example: `yarn start:theme my-theme-folder-name`
+
+If your documentation site will be uploaded to a subdirectory (ie. example.com/design-system), you can set its root path by passing the `--root` option. For example: `yarn build:theme --root design-system`
+
 
 ## Contact
 
-To contact the CMS Design System product owners, please email `WPMG_Web@cms.hhs.gov`
\ No newline at end of file
+To contact the CMS Design System product owners, please email `WPMG_Web@cms.hhs.gov`
```
Comment (line47) [inline README.md]: Is this email still the best contact for folks?

File: guides/CODING-GUIDELINES.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Coding guidelines
```
Comment (line47) [inline guides/CODING-GUIDELINES.md]: I'm wondering if this file should be on the actual website? https://design.cms.gov/guidelines/code-conventions/

File: guides/FAQ.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# FAQ
```
Comment (line47) [inline guides/FAQ.md]: Is this document needed?

File: guides/PATTERN-LIFECYCLE.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
+# Pattern lifecycle
```
Comment (line47) [inline guides/PATTERN-LIFECYCLE.md]: Still need to review this document

File: guides/SITE-PACKAGES-AND-THEMES.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
+# Site packages and themes
```
Comment (line47) [inline guides/SITE-PACKAGES-AND-THEMES.md]: Is there documentation missing here?

**Comment by benkutil** [comment]
I would suggest not moving the guides over whole-cloth. We have some things in there that we would not want to publish publically… such as the NPM publish process, etc.

**Comment by line47** [comment]
Thanks @benkutil for the feedback and concern. 

All the wiki pages are public and have been since 2017. 

The NPM publish process doesn't pose any sort of risk as further access is required to publish anything. The other wiki pages contain helpful information for contributors or anyone interested in learning more about the design system processes. I'd recommend they stay in public space.

Are their other pages in the wiki you are concerned about?

File: README.md (hunk 1 above)
Comment (line47) [reply README.md]: This is not the best email and it was suggested that a screendoor contact form be set up

**Comment by line47** [comment]
This PR is no longer relevant, closing

**Pull request closed without merging**